docs/content/stable/drivers-orms/go/yb-pgx.md

```diff
@@ -79,17 +79,15 @@ To install the package locally, run the following commands:
 mkdir yb-pgx
 cd yb-pgx
 go mod init hello
-go get github.com/yugabyte/pgx/v4
-go get github.com/yugabyte/pgx/v4/pgxpool # Install pgxpool package if you write your application with pgxpool.Connect().
 ```
 
 Optionally, you can choose to import the pgxpool package instead. Refer to [Use pgxpool API](../../../reference/drivers/go/yb-pgx-reference/#use-pgxpool-api) to learn more.
 
 ### Step 2: Set up the database connection
 
-Go applications can connect to the YugabyteDB database using the `pgx.Connect()` and `pgxpool.Connect()` functions. The `pgx` package includes all the common functions or structs required for working with YugabyteDB.
+Go applications can connect to the YugabyteDB database using the `pgx.Connect()` and `pgxpool.New()` functions. The `pgx` package includes all the common functions or structs required for working with YugabyteDB.
```

Comment:
The updates made to this page should also be done to its preview version right ? https://deploy-preview-21900--infallible-bardeen-164bc9.netlify.app/preview/drivers-orms/go/yb-pgx/

Reply:
same this for the reference page as well: https://deploy-preview-21900--infallible-bardeen-164bc9.netlify.app/preview/reference/drivers/go/yb-pgx-reference/